lms/templates/verify_student/face_photo_step.underscore

```diff
@@ -1,4 +1,18 @@
 <div id="wrapper-facephoto" class="wrapper-view block-photo face-photo-step">
+
+    <div class="requirements">
+        <h3 class="title"><%- gettext( "What You Need for Verification" ) %></h3>
+        <div class="requirement">
+            <h4 class="title"><%- gettext( "Webcam" ) %></h4>
+            <p class="copy"><%- gettext( "Make sure you have a webcam on your computer and allow camera access when the browser asks." ) %></p>
```

Comment:
You need a computer that has a webcam. When you receive a browser prompt, make sure that you allow access to the camera.